extra/migrations/27032015000000_migrate_monolithic_config.rb

```diff
@@ -0,0 +1,112 @@
+#
```

Comment:
Both monotonically increase, don't think it makes a difference (I find the "normal" format a bit easier on the eyes though).